modules/candidate_list/jsx/candidateListIndex.js

```diff
@@ -92,7 +102,7 @@ class CandidateListIndex extends Component {
    * @return {*} a formated table cell for a given column
    */
   formatColumn(column, cell, row) {
-    if (column === 'PSCID' && this.props.hasPermission('access_all_profiles')) {
+    if (column === 'PSCID') {
```

Comment:
way to sneak this one in jess haha should this change be part of this PR? 😛

Reply:
ohh nevermind, i just read that you did indeed mean to include this in the PR

Reply:
I talked to Samir yesterday and he approved this change.